### ISSUE TITLE
Bug: mycelial  start --config=config.toml seems to ignore the password from the config file

### ISSUE THREAD

**Issue by mindreframer** [mycelial/mycelial#149]
```
$ cat config.toml
[node]
display_name = "mycelial-test"
unique_id = "daemon-be1be2fe-598e-4511-af67-d6f36af71f3d"
storage_path = "daemon.db"
auth_token = "supersecret"

[server]
endpoint = "http://localhost:7777"
----------



$ mycelial  start --config=config.toml
Starting Mycelial Control Plane...
✔ Security Token: · ******** <- supersecret

Mycelial Control Plane failed to start!
check control_plane.log for more information
Starting daemon with config file config.toml...
daemon started!
```

The process is running: "  501 19049     1   0  9:37PM ttys012    0:00.08 ./myceliald --config config.toml"

but there is no server listening on 7777

```bash
mycelial|⇒ curl localhost:7777
curl: (7) Failed to connect to localhost port 7777 after 0 ms: Couldn't connect to server
```

Why is there an error happening?

**Comment by mindreframer** [comment]
OK, from the docs it seems that I have to run it with `mycelial start --daemon` to start the daemon. 

The plane is still not starting: 


```bash
mycelial start -c
error killing control plane pid 20046
deleted control plane pid file (/Users/roman/.mycelial/control_plane.pid)
Starting Mycelial Control Plane...
✔ Security Token: · ********
Mycelial Control Plane failed to start!
check control_plane.log for more information


$ cat control_plane.log
Error: AppError { status_code: 500, err: error: the following required arguments were not provided:
  --database-url <DATABASE_URL>

Usage: server --token <TOKEN> --database-url <DATABASE_URL>

For more information, try '--help'.
 }
```

Trying to provide the database URL fails: 

```bash 

$  mycelial start -c --database-url=remote.db
error: unexpected argument '--database-url' found

Usage: mycelial start <--daemon|--control-plane|--config <CONFIG>>

For more information, try '--help'.
```

**Comment by dmzmk** [comment]
Hey @mindreframer, thanks for submitting issue, it's been resolved in v0.7.0

**Comment by mindreframer** [comment]
Thanks, will check it out!
